react/week1/todolist/src/TodoList.js

```diff
@@ -0,0 +1,17 @@
+export const todos = [
```

Comment:
For the files which is not a React component and it is not returning a JSX, you don't need to use Uppercase for the first letter of the file name. `todoList` can also be abit confusing with other components. Maybe `todosCollection` ?